### PR TITLE
[CNTO-7886] tags support in Select  and [CNTO-7593] only dropdown visible

### PR DESCRIPTION
**Pull request by malay2**
[CNTO-7886](https://plivo-team.atlassian.net/browse/CNTO-7886)
tags support in Select

[CNTO-7886]: https://plivo-team.atlassian.net/browse/CNTO-7886?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

and 

For GroupAndSearchDropdown added a logic to only show dropdown and not textfield 
image

### CODE REVIEW

File: src/components/GroupAndSearchDropdown/groupAndSearchDropdown.scss [hunk 1]
```diff
@@ -44,10 +44,16 @@
   }
 
   .group-option {
-    border-bottom: 0.5px solid #E6E7EA;
+    border-bottom: 0.5px solid #e6e7ea;
```
Comment (samuellawerentz) [inline src/components/GroupAndSearchDropdown/groupAndSearchDropdown.scss:47]: Use variable here.

**Comment by samuellawerentz** [comment]
@CodiumAI-Agent /review

**Review by samuellawerentz — CHANGES_REQUESTED** [review]

**Comment by samuellawerentz** [comment]
@CodiumAI-Agent /improve

**Review by samuellawerentz — APPROVED** [review]

File: src/components/GroupAndSearchDropdown/index.js [hunk 2]
```diff
@@ -26,9 +26,11 @@ export const GroupAndSearchDropdown = ({
   onChange,
   dropdownIcon,
   openOnTextboxClick,
+  onlyDropdown,
+  parentTextFieldRef,
   ...props
 }) => {
-  const [showDropdown, setShowDropdown] = useState(false)
+  const [showDropdown, setShowDropdown] = useState(onlyDropdown)
```
Comment (rishabhPlivo) [inline src/components/GroupAndSearchDropdown/index.js:33]: do we need a default value for onlyDropdown?

File: src/components/GroupAndSearchDropdown/index.js [hunk 3]
```diff
@@ -73,7 +75,9 @@ export const GroupAndSearchDropdown = ({
                   key={childIndex}
                   onClick={() => {
                     const finalValue = `{{${child?.value}}}`
-                    textFieldRef.current.focus()
+                    if (onlyDropdown && parentTextFieldRef) parentTextFieldRef.current?.focus()
```
Comment (rishabhPlivo) [inline src/components/GroupAndSearchDropdown/index.js:78]: Can we move this to a method?

File: src/components/GroupAndSearchDropdown/index.js [hunk 4]
```diff
@@ -106,47 +118,50 @@ export const GroupAndSearchDropdown = ({
...
-              setShowDropdown(openDropdown)
-            }}
-            onChange={(e) => {
-              // So that the event is not lost
-              e.persist()
-              const caretStart = e.target.selectionStart
-              const caretEnd = e.target.selectionEnd
-              onChange(e)
+        {onlyDropdown ? (
+          <div></div>
```
Comment (rishabhPlivo) [inline src/components/GroupAndSearchDropdown/index.js:122]: can we invert the condition and return null in else case?

**Pull request merged**